tasks/main.yml

```diff
@@ -1,21 +1,24 @@
 
-- name: fail if os family is not redhat
-  fail: 
-    msg: Current role does not support on {{ ansible_os_family | lower }}, It only supports CentOS/RedHat hosts.
-  when: (ansible_os_family | lower) != "redhat"
+- name: fail if os family is not redhat or debian
```

Comment:
Why do you want to fail here? To make sure we are not running on wrong host?

Reply:
yes. i didn't want to remove the task entirely. 
so i just made it so it will now include also debian.